### DISCUSSION TITLE
Allow to also block user when submitting an abuse report

### DISCUSSION THREAD

**Discussion by darkred** [community/community#86648] [Other Feature Feedback, Questions, & Ideas]
Currently you can either block a user or report abuse _(see first two screenshots)_ .

My suggestion is to also be able to do both, i.e. to also block a user when submitting an abuse report. This could be done by adding in the abuse report form _(see last screenshot)_ :
- either a checkbox named `Also block user` (ticked by default) next to the "Submit form" button,
- or a 2nd submit button `Submit form & block user` after the existing "Submit form" one.

<details>
<summary>Screenshots:</summary>

 
![2024-01-12_121138](https://github.com/community/community/assets/723651/a64eb545-c1eb-439a-a9fd-18cd5f1f80be) ![2024-01-12_000649](https://github.com/community/community/assets/723651/ca58b68d-2c5f-4c40-bd31-0eb2420deb17)

&nbsp;
![2024-01-12_001430](https://github.com/community/community/assets/723651/2cc26512-5845-4f5e-a7e6-bfd5580b3689)
</details>

**Comment by mishmanners** [comment]
I've tagged this as feedback and the Product team can take a look at this for future updates.

**Reply by darkred** [reply]
_(Off-topic)_ @mishmanners Could you also please change the label of the [other](https://github.com/orgs/community/discussions/81390) discussion I have , from 'General' to: `Issues`, `Pull Requests` and `Product Feedback` ? Thank you very much!

**Reply by mishmanners** [reply]
Yep can do. And that's done for you.
